### PR TITLE
py-sybil: Update to version 3.0.0; Add Python 39; Remove Python 35 36

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

  macOS 11.6.1 20G224 x86_64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by kurthindenburg** [comment]
Is there any specific reason to move to github?  Typically we would use the github PG.  Also, whitespace changes should be a separate commit.

**Review by mascguy — APPROVED** [review]

**Review by mascguy — APPROVED** [review]

**Comment by mascguy** [comment]
> Is there any specific reason to move to github? Typically we would use the github PG. Also, whitespace changes should be a separate commit.

Given that the whitespace changes aren't too extensive, I'm okay with it as-is. But up to you.

**Comment by mascguy** [comment]
> Is there any specific reason to move to github?

Steve, is there a reason we'd want to use GitHub, rather than PyPi, for this particular port?

**Comment by mascguy** [comment]
> Is there any specific reason to move to github? Typically we would use the github PG.

Well, we're not really moving to GitHub per se, as the homepage is the only thing referencing GitHub. Otherwise, the port is still using PyPi for both source archives, as well as the livecheck.

Kurt, does that alleviate your concerns?

File: python/py-sybil/Portfile [hunk 1]
```diff
@@ -17,31 +17,26 @@ long_description    This library provides a way to test examples in your \
...
 
-checksums           rmd160  5fa7b4e6b9fba4dd807bfbf4850b804be8caa016 \
-                    sha256  597d71e246690b9223c132f0ed7dcac470dcbe9ad022004a801e108a00dc3524 \
-                    size    29316
+checksums           rmd160  c4a72e2d29f973ca006d36812bc81c22d522a326 \
+                    sha256  7692ed66e793e5e79ae6a70cf2cf861917ed48eaff0d8adf825e64d85820f251 \
+                    size    38011
 
-python.versions     27 35 36 37 38
+python.versions     27 37 38 39
```
Comment (reneeotten) [inline python/py-sybil/Portfile:26]: this new version doesn't support PY27 anymore; please account for this (and check this for the other PRs you submitted as well)

**Review by reneeotten — CHANGES_REQUESTED** [review]

**Comment by reneeotten** [comment]
> Is there any specific reason to move to github? Typically we would use the github PG. Also, whitespace changes should be a separate commit.

this is okay, the removed lines are the defaults set by the `python` PG for use with PyPI

**Comment by kurthindenburg** [comment]
I won't object to this

**Comment by mascguy** [comment]
> I won't object to this

Great, thanks for the feedback Kurt!

Steve, once you make the change requested by Renee, we'll be able to merge this.

File: python/py-sybil/Portfile (hunk 1 above)
Comment (essandess) [reply python/py-sybil/Portfile]: Thank you. Done.

**Pull request merged**